contracts/TopShotShardedCollection.cdc

```diff
@@ -35,8 +35,8 @@
 
 */
 
-import NonFungibleToken from 0xNFTADDRESS
-import TopShot from 0xTOPSHOTADDRESS
+import NonFungibleToken from 0x631e88ae7f1d7c20
```

Comment:
Are these contract aliases intentionally replaced? Looks like other contracts in this repo have aliases

Reply:
That was my fault @sisyphusSmiling, i just fixed it